### PR TITLE
New list option `--include-injected` shows injected packages in main apps

### PR DESCRIPTION
**Pull request by PratikBhusal**
The main reason for this pull request is to have the ability to see the injected packages without having to examine the underlying `pipx_metadata.json` file stored in each virtual environment. The name is a work in progress so I would like to get some suggestions about what to do with that.

If there is anything else that could be a cause for concern, please let me know. Thank you.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ dev
 - [refactor] Moved all commands to separate files within the commands module (#255).
 - [bugfix] Continue reinstalling packages after failure
 - [bugfix] Hide cursor while pipx runs
+- [feature] `list` now has a new option `--include_injected` to show the injected packages in the main apps
```
Comment (cs01) [inline docs/changelog.md:9]: `--include-injected` (replace underscore)

File: src/pipx/main.py [hunk 2]
```diff
@@ -388,6 +388,11 @@ def _add_list(subparsers):
         help="List installed packages",
         description="List packages and apps installed with pipx",
     )
+    p.add_argument(
+        "--include-injected",
+        action="store_true",
+        help="show the injected packages in the main apps",
```
Comment (cs01) [inline src/pipx/main.py:394]: Show packages injected into the main app's environment

File: src/pipx/commands/common.py [hunk 3]
```diff
@@ -175,6 +181,10 @@ def _get_list_output(
         output.append(
             f"    - {red(name)} (symlink missing or pointing to unexpected location)"
         )
+    if injected_package_names:
+        output.append(f"    {bold('Injected Packages')}")
```
Comment (cs01) [inline src/pipx/commands/common.py:185]: Remove bold

**Comment by PratikBhusal** [comment]
@cs01 I have implemented your requested changes. Please feel free to look at it again.

**Comment by cs01** [comment]
Looks good besides merge conflicts

**Comment by cs01** [comment]
Thanks for the update. I am out of town at the moment but will review it when I’m back next week.

File: docs/changelog.md [hunk 4]
```diff
@@ -15,6 +15,7 @@ dev
 - [bugfix] Hide cursor while pipx runs
 - [feature] Add environment variable `USE_EMOJI` to allow enabling/disabling emojies (#376)
 - [refactor] Moved all commands to separate files within the commands module (#255).
+- [feature] `list` now has a new option `--include-injected` to show the injected packages in the main apps
```
Comment (cs01) [inline docs/changelog.md:18]: This should be moved to the `dev` section since it was not included in a release. Otherwise, this looks good.

Comment (PratikBhusal) [reply docs/changelog.md]: I have moved it to the correct position.

**Pull request merged**